test/postgresql.test.js

```diff
@@ -206,6 +206,15 @@ describe('postgresql connector', function () {
           console.warn.restore();
         });
 
+        it('should work', function(done) {
+          Post.find({where: {content: {regexp: '^a/i'}}}, function(err, posts) {
```

Comment:
ok, I wonder if we want to be strict, i.e., the string should be:

1. /pattern/
2. /pattern/i
3. pattern

I'm fine to allow `pattern/i` if it doesn't cause confusions.